quantum/process_keycode/process_ucis.c

```diff
@@ -90,6 +90,7 @@ void register_ucis(const char *hex) {
 
 bool process_ucis (uint16_t keycode, keyrecord_t *record) {
   uint8_t i;
+  get_unicode_input_mode();
```

Comment:
This line on its own wouldn't make sense to someone reading this bit of code. When I see a `get_unicode_input_mode()` function call, I'd expect the returned value to be used / stored in a variable; otherwise, why would you be getting the input mode? So the function is being used here due to its side-effect, which isn't ideal, and certainly isn't clear.

I suggest moving the initial read from EEPROM to a function called `prepare_unicode_input_mode()` (or something similar), and then calling it here, as well as in `get_unicode_input_mode()`. That way it's clear that this call is potentially mutating some state.